### DISCUSSION TITLE
Are Shorts included by default?

### DISCUSSION THREAD

**Discussion by cbunn81** [jmbannon/ytdl-sub#583] [Q&A] [answered]
Hi there. First, thanks for such an amazing tool.

I've been downloading some channels, and the regular videos have been downloading just fine, but it seems that the Shorts from those channels aren't downloading at all. There don't appear to be any errors or attempts to download them in the logs.

Perhaps there is some setting I need to enable to ensure that they are included?

Thanks!

**Comment by jmbannon** [comment, marked as answer]
I think it depends on the URL you use - this is yt-dlp behavior. You can add multiple URLs to a TV show like this:
https://ytdl-sub.readthedocs.io/en/latest/presets.html#usage (see second usage example)
